src/dialect/mod.rs

```diff
@@ -1136,6 +1142,11 @@ pub trait Dialect: Debug + Any {
     fn supports_notnull_operator(&self) -> bool {
         false
     }
+
+    /// Returns true if the dialect supports parsing statements without a semicolon delimiter.
```

Comment:
Would it be possible to give an example here? 

Something like

```suggestion
    /// Returns true if the dialect supports parsing statements without a semicolon delimiter.
    /// 
    /// If returns true, the following SQL will not parse. If returns `false` the SQL will parse
    ///
    /// ```sql
    /// SELECT 1
    /// SELECT 2
    /// ```
```

Reply:
Done 👍